CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 - Protocol validation for source map image type. ([#1869](https://github.com/getsentry/relay/pull/1869))
 - Add PHP support for profiling. ([#1871](https://github.com/getsentry/relay/pull/1871))
 
+**Bug Fixes**:
+
+- Enforce rate limits for session replays. ([#1877]https://github.com/getsentry/relay/pull/1877)
```

Comment:
```suggestion
- Enforce rate limits for session replays. ([#1877](https://github.com/getsentry/relay/pull/1877))
```

Reply:
Thanks. I should not write markup manually.